### PR TITLE
chore: send analytics data on all navigation ends

### PR DESCRIPTION
**Pull request by josephperrott**
Fixes https://github.com/angular/material2/issues/10022

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -24,16 +24,20 @@
...
   <script>
-    window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
-    ga('create', 'UA-8594346-24', 'auto');
-    ga('send', 'pageview');
+  // Note this is a customised version of the GA tracking snippet
+  // See the comments below for more info
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;
+  ~i.name.indexOf('NG_DEFER_BOOTSTRAP')|| // only load library if not running e2e tests
```
Comment (IgorMinar) [inline src/index.html]: e2e tests run on local host or other non-prod host, this causes all the GA pings to be ignored/filtered out automatically. In other words, it's typically not necessary to treat e2e tests differently.

**Comment by IgorMinar** [comment]
has this change been deployed? I don't see any pings from material.angular.io yet. thanks for making this change @josephperrott

**Comment by josephperrott** [comment]
@IgorMinar This is now pushed, our first release of [material.angular.io](material.angular.io) with this change was on Feb 27 around 10am PST, you should see data collected beginning then.

**Comment by IgorMinar** [comment]
yup. I see the data now in our reports. thanks!

**Pull request merged**